packages/addons/service/nextpvr/source/bin/nextpvr.start

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+# SPDX-License-Identifier: GPL-2.0-only
+# Copyright (C) 2021-present Team LibreELEC (https://libreelec.tv)
+
+. /etc/profile
+oe_setup_addon service.nextpvr
+
+# check if nextpvr-server is already successful installed
+if [ ! -f "${ADDON_DIR}/extract.ok" ]; then
+  cd ${ADDON_DIR}
+  nextpvr-downloader
+fi
+
+export NEXTPVR_DATADIR_USERDATA=${ADDON_HOME}/config/
+export NEXTPVR_DVBDIR=${ADDON_DIR}/dtv-scan-tables/
+
+satip=$(xmlstarlet select -t -v /settings/setting[@id=\"satiprtsp\"]  ${ADDON_HOME}/settings.xml)
+if [ -n "$satip" ]; then
+  export SATIP_RTSP_PORT=$satip
+fi
+
+waitfor=$(xmlstarlet select -t -v /settings/setting[@id=\"waitfor\"]  ${ADDON_HOME}/settings.xml)
+if [ -n "$waitfor" ]; then
+  read -d. uptime < /proc/uptime
+  while [ $uptime -lt $waitfor ]; do
+    sleep 1
+    read -d. uptime < /proc/uptime
+  done
+fi
```

Comment:
```suggestion
sleep "$waitfor"
```

it should just add a "sleep" timeout ?
we have the same at TVH too, its much easier
https://github.com/LibreELEC/LibreELEC.tv/blob/master/packages/addons/service/tvheadend43/source/bin/tvheadend43.start#L85-L88

Reply:
I don't like the the fix in TVH because it is not based on uptime.

Reply:
not sure I understand the purpose of the wait here, normally you want to delay a start for x seconds as workaround for some stuff, so it does not really matter if you wait 5sec with uptime or 4 secs with a regular sleep

Because uptime is even more inaccurate at slower systems, you can have a uptime of 10secs but kodi is still not loaded at all. If you want to load the addon after kodi is started you can easily change it at the systemd file and add there kodi as dep.

Reply:
Looking at dmesg you can generally tell the uptime when devices are loaded and drives are on line.
```
-- Journal begins at Tue 2022-10-11 22:53:25 CDT, ends at Wed 2022-10-12 06:21:53 CDT. --
...
Oct 11 22:53:31.688549 KODI systemd[1]: Started NextPVR Server.
...
Oct 11 22:53:42.858665 KODI kernel: saa7164 0000:04:00.0: DVB: registering adapter 1 frontend 0 (Samsung S5H1411 QAM/8VSB Frontend).
```
To me it is more logical to tell the user to set the wait to 20 seconds or so (from startup) then 12 seconds from when the service would have been initiated.  The other advantage is there is no wait when the server is restarted well after boot